en/services/mission.md

```diff
@@ -311,6 +311,76 @@ Note:
 The *defacto* standard file format for exchanging missions/plans is discussed in: [File Formats > Mission Plain-Text File Format](../file_formats/README.md#mission_plain_text_file).
 
 
+## Mission Command Detail
+
+This section is for clarifications and additional information about common mission items.
+In particular it is intended for cases that are difficult to document in the specification XML, or when images will much better describe expected behaviour.
+
+
+### Loiter Commands (`MAV_CMD_NAV_LOITER_*`) {#loiter_commands}
+
+Loiter commands are provided to allow a vehicle to hold at a waypoint based on time, number of turns, or indefinitely.
```

Comment:
, or to allow the vehicle to climb/descend at this position (if we also want to add MAV_CMD_NAV_LOITER_TO_ALT)

Reply:
Yes we do. Thanks!